backend/presentation/services.py

```diff
@@ -0,0 +1,5 @@
+class CommentService:
+    def update_comment(self, comment, validated_data):
+        comment.text = validated_data.get("text", comment.text)
+        comment.save()
```

Comment:
```suggestion
        comment.save(update_fields=["text"])
```
It is a good practice to specify `update_fields` for better performance.